packages/non-core/coffeescript-compiler/package.js

```diff
@@ -0,0 +1,23 @@
+Package.describe({
+  name: 'coffeescript-compiler',
+  summary: 'Compiler for CoffeeScript code, supporting the coffeescript package',
+  // This version of NPM `coffeescript` module, with _1, _2 etc.
+  // If you change this, make sure to also update ../coffeescript/package.js to match.
+  version: '1.12.7_1'
+});
+
+Npm.depends({
+  'coffeescript': '1.12.7',
+  'source-map': '0.5.6'
+});
+
+Package.onUse(function (api) {
+  api.use('babel-compiler@6.19.4');
+  api.use('ecmascript@0.8.2');
+
+  api.mainModule(['coffeescript-compiler.js'], 'server');
```

Comment:
Updated. I have “allow edits from maintainers” checked, so I’d think you should be able to push commits . . .